app/policies/manual_policy.rb

```diff
@@ -0,0 +1,11 @@
+class ManualPolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  alias_method :new?, :index?
+  alias_method :create?, :index?
+  alias_method :edit?, :index?
+  alias_method :update?, :index?
+  alias_method :show?, :index?
```

Comment:
If the intent is to allow all these actions by default, would something like

```ruby
[:index?, :new?, :create?, :edit?, :update?, :show?].each do |allowed_action|
  define_method allowed_action do
    true
  end
end
```
communicate the same intent without indicating via `alias_method` that the methods are semantically equivalent (and that `:index?` has some kind of primary status)?